x-pack/auditbeat/module/system/package/faltbuffers.go

```diff
@@ -0,0 +1,191 @@
+// Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
Wouldn't changing the path, make git track all the package files as new changes ? Would it be better to do that as a separate PR as it would be difficult to track the changes of the current PR ?